pallets/gear/src/tests.rs

```diff
@@ -7239,6 +7243,13 @@ mod utils {
         CodeId::generate(code).into()
     }
 
+    /// Gets next message id, but doesn't remain changed the state of the nonces
```

Comment:
seems can be removed as copypaste from bench test utils

Reply:
done.